### PR TITLE
Document `ReflectionGenerator::isClosed()`

### PR DESCRIPTION
**Pull request by TimWolla**

### CODE REVIEW

File: reference/reflection/reflectiongenerator/isclosed.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="ReflectionGenerator">
+   <modifier>public</modifier> <type>bool</type><methodname>ReflectionGenerator::isClosed</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Returns whether the execution reached the end of the function, a return statement or if an exception was thrown.
+  </para>
```
Comment (Girgias) [inline reference/reflection/reflectiongenerator/isclosed.xml:17]:
```suggestion
  <simpara>
   Returns whether the execution reached the end of the function, a return statement or if an exception was thrown.
  </simpara>
```

File: reference/reflection/reflectiongenerator/isclosed.xml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns whether the generator finished executing.
+  </para>
```
Comment (Girgias) [inline reference/reflection/reflectiongenerator/isclosed.xml:29]:
```suggestion
  <simpara>
   Returns whether the generator finished executing.
  </simpara>
```

File: reference/reflection/reflectiongenerator/isclosed.xml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+ </refsect1>
+
+  <refsect1 role="examples">
+  &reftitle.examples;
+  <para>
+   <example>
+    <title><methodname>ReflectionGenerator::isClosed</methodname> example</title>
```
Comment (Girgias) [inline reference/reflection/reflectiongenerator/isclosed.xml:36]: The wrapping para tag is useless here

**Review by Girgias — APPROVED** [review]
LGTM, minor markup remarks

File: reference/reflection/reflectiongenerator/isclosed.xml (hunk 3 above)
Comment (TimWolla) [reply reference/reflection/reflectiongenerator/isclosed.xml:36]: FWIW: I've used `ReflectionGenerator::getThis()` as the basis here.

Comment (cmb69) [reply reference/reflection/reflectiongenerator/isclosed.xml:36]: Doing mass changes for this is not a good idea regarding translations, but additions should better not add useless elements.

File: reference/reflection/reflectiongenerator/isclosed.xml (hunk 1 above)
Comment (cmb69) [reply reference/reflection/reflectiongenerator/isclosed.xml:17]: Only general DocBook documentation; in short, `<simpara>` resembles a HTML `<p>`, while `<para>` allows block level children, and is more like a HTML `<div>`.

File: reference/reflection/reflectiongenerator/isclosed.xml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+a
+bool(false)
+a
+bool(false)
+bool(true)
+]]>
+    </screen>
+   </example>
+  </para>
+ </refsect1>
```
Comment (Girgias) [inline reference/reflection/reflectiongenerator/isclosed.xml:72]:
```suggestion
  <example>
   <title><methodname>ReflectionGenerator::isClosed</methodname> example</title>
   <programlisting role="php">
<![CDATA[
<?php

function gen()
{
    yield 'a';
    yield 'a';
}

$gen = gen();
$reflectionGen = new ReflectionGenerator($gen);

foreach ($gen as $value) {
    echo $value, PHP_EOL;
    var_dump($reflectionGen->isClosed());
}

var_dump($reflectionGen->isClosed());

?>
]]>
   </programlisting>
   &example.outputs;
   <screen>
<![CDATA[
a
bool(false)
a
bool(false)
bool(true)
]]>
   </screen>
  </example>
 </refsect1>
```

**Pull request merged**